ui/app/components/policy-form.hbs

```diff
@@ -38,7 +38,7 @@
           </ToolbarActions>
         </Toolbar>
         {{#if this.showFileUpload}}
-          <TextFile @inputOnly={{true}} @file={{this.file}} @onChange={{this.setPolicyFromFile}} />
```

Comment:
there both inputs, so this naming confused me a little bit. Open to alternatives! I'd also tried out `uploadOnly` but wasn't sure if that was clear enough

Reply:
`hideTextAreaToggle` actually hides the entire label right? With that in mind I think `uploadOnly` makes the most sense

Reply:
ah yes! it does! sounds good 👍